### ISSUE TITLE
Is it distributed?

### ISSUE THREAD

**Issue by rizplate** [joblib/joblib#528]
Can it run jobs on different machines?

If yes, where is the data stored? some remotely accessed database? or filesystem?

**Comment by lesteve** [comment]
You asked the very same question to a few different projects. Unfortunately open-source projects do not have the resource to answer generic questions like this one. I am afraid I am going to close this one.

**Comment by rizplate** [comment]
so what is wrong with me asking question to other projects? what is it a problem for you ? I can ask as many question to other projects.

since it is an open source project, who are you to close this question? 

If you don't know, let others answer it

**Comment by GaelVaroquaux** [comment]
@lesteve knows very well the answer to the question. He is one of the
maintainers of the project.

The problem with you asking many questions to many projects in parallel
on their issue tracker is that for each project you are taking time to
core developers. As the tracker is a tool to report problems with the
software to the core developers. The trackers become flooded in support
questions. Answering them, even with a short text takes time from core
developers. Maybe it's only a few minutes, but these add up hugely. As a
result, core developers have less and less time. Less time to work on the
project, which is what they have volunteered to do, but also less free
time in general.

Open source is free. Nobody is payed to answer your questions. You get
the software for free, and you do what you may with it.

Another issue is the phrasing of your questions, which seem very
impersonal and some may read as passive-aggressive. It does not help
getting the best reaction from most people. When I read your last answer,
I was annoyed, I must confess.
